path: [ [0,0], [1, 0], [1, 1], [0, 1]]
step_size: .1
path_frame: odom
odom_frame: odom
